DOC-1646 - Replace screenshots and update content in Prometheus Integration article (#2179)
RedisLabs/redislabs-docs: content/rs/clusters/monitoring/prometheus-integration.md

- Display Redis Enterprise Software metric data alongside data from other applications

> {{<image filename="/images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster and Grafana queries those metrics for visualization.">}}{{< /image >}}

Reviewer: Broken image. I'm not sure if the forward slash is the problem, but that's the main thing that stood out to me as different from the existing image shortcodes.
```suggestion
{{<image filename="images/rs/grafana-prometheus.png" alt="Graphic showing how Prometheus and Grafana collect and display data from a Redis Enterprise Cluster. Prometheus scrapes metrics from the Redis Enterprise cluster and Grafana queries those metrics for visualization.">}}{{< /image >}}
```

                    - prometheus-server:prometheus
        ```

>     1. To start the containers, run: `docker compose up -d`

Reviewer: Maybe separate the `docker compose` command into a code block on the next line.

    1. To check that all the containers are up, run: `docker ps`

>     1. In your browser, login to Prometheus at http://localhost:9090 to make sure the server is running.

Reviewer: ```suggestion
    1. In your browser, sign in to Prometheus at http://localhost:9090 to make sure the server is running.
```

    1. Select **Status** and then **Targets** to check that Prometheus is collecting data from the Redis Enterprise cluster.

>         {{<image filename="/images/rs/prometheus-target.png" alt="The Redis Enterprise target showing that Prometheus is connected to the Redis Enterprise Cluster.">}}{{< /image >}}

Reviewer: Broken image.

        If Prometheus is connected to the cluster, you can type **node_up** in the Expression field on the Prometheus home page to see the cluster metrics.

1. Configure the Grafana datasource:

>     1. Login to Grafana. If you installed Grafana locally, go to http://localhost:3000 and login with:

Reviewer: ```suggestion
    1. Sign in to Grafana. If you installed Grafana locally, go to http://localhost:3000 and sign in with:
```

                    - prometheus-server:prometheus
        ```

    1. To start the containers, run:

>     ```sh
>     $ docker compose up -d
>     ```

Reviewer: I think you might need to indent this code block so that the steps don't start over from `a`. Currently, the steps go `a, b, a, b, c`.
```suggestion
        ```sh
        $ docker compose up -d
        ```
```

    1. To check that all the containers are up, run: `docker ps`
    1. In your browser, sign in to Prometheus at http://localhost:9090 to make sure the server is running.
    1. Select **Status** and then **Targets** to check that Prometheus is collecting data from the Redis Enterprise cluster.

        {{<image filename="images/rs/prometheus-target.png" alt="The Redis Enterprise target showing that Prometheus is connected to the Redis Enterprise Cluster.">}}{{< /image >}}

        If Prometheus is connected to the cluster, you can type **node_up** in the Expression field on the Prometheus home page to see the cluster metrics.

1. Configure the Grafana datasource:
    1. Sign in to Grafana. If you installed Grafana locally, go to http://localhost:3000 and sign in with:

        - Username: admin
        - Password: secret

    1. In the Grafana configuration menu, select **Data Sources**.

    1. Select **Add data source**.

    1. Select **Prometheus** from the list of data source types.

>     {{<image filename="images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}

Reviewer: The indentation of this screenshot might be causing a similar issue with the steps: `a, b, c, d, a`.
```suggestion
        {{<image filename="images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}
```

    1. In the Grafana configuration menu, select **Data Sources**.

    1. Select **Add data source**.

    1. Select **Prometheus** from the list of data source types.

>     {{<image filename="/images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}

Reviewer: Broken image.

    1. Enter the Prometheus information:

        - Name: `redis-enterprise`
        - URL: `http://<your prometheus address>:9090`
        - Access: `Server`

>         {{<image filename="/images/rs/prometheus-connection.png" alt="The Prometheus connection form in Grafana.">}}{{< /image >}}

Reviewer: Broken image.
